UnitTestFancyZonesEditor/GridLayoutModelTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using FancyZonesEditor.Models;
+
+namespace UnitTestFancyZonesEditor;
```

Comment:
Different namespaces in `GridLayoutModelTests` and `DefaultLayoutssModelTests`, let's keep one of them.